embedded/store/tx.go

```diff
@@ -441,7 +441,7 @@ func (tx *Tx) readFrom(r *appendable.Reader) error {
 				return err
```

Comment:
(commend for few lines above) Post-merge: We don't check if `mdLen <= maxKVMetadataLen`, this could cause out-of-bounds panic in line 439 (`mdbs[:mdLen]`)